### PR TITLE
Enable inspection of config-files for Ubuntu hosts

### PR DESCRIPTION
**Pull request by tgoettlicher**

### CODE REVIEW

File: lib/managed_files_database.rb [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      end
+    end
+
+    @changed_files = result
+  end
+
+  def parse_changes_line(line)
+    # rpm provides lines per config file where first 9 characters indicate which
+    # properties of the file are modified
+    @rpm_changes, *fields = line.split(" ")
```
Comment (mauromorales) [inline lib/managed_files_database.rb]: maybe we can rename this so it doesn't imply it's aware of `rpm`

File: spec/unit/rpm_database_spec.rb [hunk 2]
```diff
@@ -18,245 +18,59 @@
...
-    end
-
-    it "it adds other_rpm_changes in case of an unexpected new/additional rpm tag" do
-      line = "S........N  c /etc/pulse/client.conf"
-      file, changes, flag = subject.parse_rpm_changes_line(line)
-      expect(file).to eq("/etc/pulse/client.conf")
-      expect(changes).to match_array(["size", "other_rpm_changes"])
-      expect(flag).to eq("c")
-    end
+  describe "#managed_file_list" do
```
Comment (mauromorales) [inline spec/unit/rpm_database_spec.rb]: maybe `managed_files_list` ?

File: lib/managed_files_database.rb (hunk 1 above)
Comment (tgoettlicher) [inline lib/managed_files_database.rb]: dpkg prints rpm format. That's why it's called @rpm_changes.

Comment (mauromorales) [inline lib/managed_files_database.rb]: fair enough ;)

**Comment by tgoettlicher** [comment]
Squashed: https://github.com/SUSE/machinery/pull/1726

**Pull request closed without merging**